packages/web-components/src/customized-built-ins/image/image.ts

```diff
@@ -0,0 +1,61 @@
+import { attr, customElement, FASTElement } from '@microsoft/fast-element';
+import { ImageFit, ImageShape } from '../../image/image.options.js';
+import { styles as FluentImageStyles } from './image.styles.js';
+
+/**
+ * The base class used for constucting a fui image customized built-in element
+ * @public
+ */
+@customElement({
+  name: `fui-image`,
```

Comment:
AFAIR the current agreement is to export customized built-ins side by side with existing web components?
If that is the case, the component name collides between wc and customized-built-ins. Is that intentional and the application should decide which one to use or do we want the name to be unique?
Also `fui` should be `FluentDesignSystem.prefix.`

Reply:
If `fui` should be the prefix that's fine - this was to *avoid* the current collision.

IF we change, it'll be another breaking change for folks currently using `fluent-`, that's fine but let's make sure we're good with it. If so, feel free to open a PR to change